boxoffice/models/item.py

```diff
@@ -44,6 +45,20 @@ class Item(BaseScopedNameMixin, db.Model):
 
     cancellable_until = db.Column(db.DateTime, nullable=True)
 
+    __roles__ = {
+        'item_owner': {
+            'write': {},
```

Comment:
Surely the item owner can write all these fields? If the `write` action is _unused_ at the moment, leave this out. Less ambiguity that way. RoleMixin will add a blank for you. (Also applies to other places `__roles__` is defined.)